### ISSUE TITLE
omni_normtest TODO ntoe

### ISSUE THREAD

**Issue by jbrockmendel** [statsmodels/statsmodels#4062]
from stats.stattools.omni_normtest
```
    # TODO: change to exception in summary branch and catch in summary()
    # behavior changed between scipy 0.9 and 0.10
```

scipy is up to at least 0.19.1.  Is this actionable?

**Comment by bashtage** [comment]
Yes, since minimum is way about 0.10.

**Comment by josef-pkt** [comment]
needs design decision, 
I don't remember what the decision was between warn and raise in hypothesis test, or whether we even have a consistent pattern.
I don't find anything in issue search (but maybe the discussion was in scipy issues or PRs)

Right now I think returning nan and warn works fine, the only internal users of it are in regression summaries.

**Comment by jbrockmendel** [comment]
So... the appropriate action is to get rid of the TODO note?

**Comment by josef-pkt** [comment]
Yes, but also to open an issue for what to do if sample size is too small to compute a test statistic.

(Right now I'm in favor of nan and warn, that's why removing todo is fine. AFAIR or AFAICS scipy stats raises.)
